assets/js/theme/common/product-details.js

```diff
@@ -443,6 +443,38 @@ export default class ProductDetails extends ProductDetailsBase {
         this.setLiveRegionAttributes($addToCartBtn.next(), 'status', 'polite');
     }
 
+    /**
+     *
+     * Add a product to wishlist
+     *
+     */
+    addProductToWishlist() {
+        const $addToWishlistBtn = $('.productView-options .form-wishlist');
```

Comment:
as far as I remember we've discussed to add more specific selector that will prevent unauthorised users from adding to wish list.

Reply:
@bc-alexsaiannyi Yes, we discussed this in the following subtask 726, but it may also apply to the current PR.

Reply:
@BC-tymurbiedukhin done!